### PR TITLE
style: :lipstick: moved some of aesthetic theme over from Sprout

### LINKED ISSUE #23: Move aesthetic theme from Sprout and convert to website theme

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

I moved some items over from the style in Sprout. We might need to look into how well the colours are for colour blindness, I was having some troubles with the purple. I also didn't use the font, it looks terrible as a website font.

On Tuesday I can go over how this looks like and how to use the seedcase-theme repo to test out the website theming.

Closes #23

### CODE REVIEW

**Comment by signekb** [comment]
Nice! Which purple (HEX or line) are you referring to?

File: _extensions/seedcase-theme/theme.scss [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+$font-family-sans-serif: "Maven Pro", sans-serif;
 $font-family-monospace: "Fira Code", monospace;
-$headings-font-family: "Poppins", sans-serif;
-$navbar-bg: #990032;
+$headings-font-family: "Noto Sans", sans-serif;
+
+/* brand colour #003224 - the one below is a slightly lighter version of the brand colour */
+$primary: #196440;
+$secondary: #C6F0D3;
+$link-color: #8b8b8b;
```
Comment (lwjohnst86) [inline _extensions/seedcase-theme/theme.scss:14]: @signekb it is this colour

Comment (signekb) [reply _extensions/seedcase-theme/theme.scss:14]: Wait, the #8b8b8b? That's grey, right?

Comment (lwjohnst86) [reply _extensions/seedcase-theme/theme.scss:14]: That's what I mean! It's a terrible colour for colour blind, I can't tell what it is!!

Comment (signekb) [reply _extensions/seedcase-theme/theme.scss:14]: Ah, alright! Definitely! I was also unsure of why you had placed the grey as the "link colour"

Comment (lwjohnst86) [inline _extensions/seedcase-theme/theme.scss:14]:
```suggestion
```

**Pull request merged**